acos_client/v21/slb/server.py

```diff
@@ -22,11 +22,12 @@ class Server(base.BaseV21):
     def get(self, name, **kwargs):
         return self._post("slb.server.search", {'name': name}, **kwargs)
 
-    def create(self, name, ip_address, **kwargs):
+    def create(self, name, ip_address, conn_limit=8000000, **kwargs):
```

Comment:
Since the conn-limit isn't required by acos we shouldn't be defaulting it when it's not specified.